gpkit/nomials/math.py

```diff
@@ -523,26 +468,27 @@ def as_posyslt1(self, substitutions=None):
             #  monomial indexes pre-substitution, and whose values are the
             #  percentage of the simplified  monomial's coefficient that came
             #  from that particular parent.
-
-            self.pmap = pmap  # pylint: disable=attribute-defined-outside-init
-            try:
-                p = Posynomial(exps, cs, simplify=False)
-            except ValueError:
+            #
+            hmap = self._simplify_posy_ineq(hmap, self.pmap)
+            if hmap is None:
+                continue
+            if all(c == 0 for c in hmap.values()):
+                continue  # skip tautological 0'd constraint
+            p = Posynomial(hmap)
+            out.append(p)
+            if p.any_nonpositive_cs:
                 raise RuntimeWarning("PosynomialInequality %s became Signomial"
                                      " after substitution %s"
                                      % (self, substitutions))
-            out.append(p)
         return out
 
     def sens_from_dual(self, la, nu):
         "Returns the variable/constraint sensitivities from lambda/nu"
         if not la or not nu:
-            # as_posyslt1 created no inequalities
-            return {}
+            return {}  # as_posyslt1 created no inequalities
         la, = la
         nu, = nu
```

Comment:
Huh? No, it's to say that there are no sensitivities because as_posyslt1 created no inequalities